connection.js

```diff
@@ -22,40 +19,33 @@ async function makeDbCalls(err){
     // Handle any DB connection errors
     if(err){
         console.error(`An error occurred while connecting to the MongoClient: ${err}`);
-        process.exit(1);
+        return {error: err};
```

Comment:
Are you actually using `{ error: err }` anywhere? I think since you've already printed the error, you're fine to just `return` here